mozilla-services/socorro
# bug 1493687: move crontabber tables to Django

This creates a new Django app named `cron` which has two tables in it. Then it reworks all the crontabber stuff to use those tables for bookkeeping.

This is an easier step than rewriting all of crontabber. The next step would be to implement basic crontabber stuff in Django and then start rewriting crontabber apps into Django cron apps. There are a couple of things we can do before we do that which will make that a lot easier, so I'm pushing that off until later.

To test this:

1. set up a dev environment which exercises many of the crontabber apps
2. run the crontabber container
3. check `/crontabber-state`
4. check out the new Django admin pages for `cron_job` and `cron_log`
5. run some other crontabber commands like the infamous `--list-jobs` and the quixotic `--reset`
